vendor-extra/JatsContentBundle/src/EventListener/BuildView/PubDatePartsTimeListener.php

```diff
@@ -15,7 +15,6 @@ final class PubDatePartsTimeListener
     private const YEAR_PATH = 'jats:year[number(.)=.][1]';
     private const MONTH_PATH = 'jats:month[number(.)=.][1]';
     private const DAY_PATH = 'jats:day[number(.)=.][1]';
-    private const PARTS_PATH = self::YEAR_PATH.'|'.self::MONTH_PATH.'|'.self::DAY_PATH;
```

Comment:
surprising this was not marked as unused by some static analysis

Reply:
Yeah. 🤔